flake.nix

```diff
@@ -80,7 +80,7 @@
             wasmBindgenVersion = assert builtins.length wasmBindgenCargoVersions == 1; builtins.elemAt wasmBindgenCargoVersions 0;
           in
           {
-            wasm-bindgen-cli = prev.wasm-bindgen-cli.override {
+            wasm-bindgen-cli = prev.wasm-bindgen-cli.overrideAttrs {
```

Comment:
Note that latest versions from Nixpkgs now provides a nice builder and remove the need for override. See https://github.com/tweag/nickel/commit/f3be2961fef41548c5f6f42d6b5b205c3c083ffd#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0L87-R90 for an example of migration